src/encoded/types/shared_biosample.py

```diff
@@ -0,0 +1,86 @@
+from .biosample import generate_summary_dictionary
+
+def biosample_summary_dictionary(request, biosampleObject=None):
```

Comment:
since this function is no longer returns only a dictionary I think it should be renamed, otherwise, it is misleading. Perhaps "biosample_summary_information" or something similar to that. I would also probably define variables with explicit names in dataset.py tp clearly distinguish between the [0] that is a dictionary and [1] that is a boolean flag.

Reply:
Used biosample_summary_information instead